### PR TITLE
WND-27678 fixed kube-proxy version, parameters configurable

### PR DESCRIPTION
**Pull request by jojand**

### CODE REVIEW

File: aws/cluster/variables.tf [hunk 1]
```diff
@@ -75,6 +75,22 @@ variable "kube-dns" {
...
+# kube-proxy
+variable "kube-proxy" {
+  type = object({
+    enabled          = bool
+    clusterCIDR      = string
+    cpuRequest       = string
+    hostnameOverride = string
+  })
+  default = {
+    enabled          = false
```
Comment (wandera-petr-cerny) [inline aws/cluster/variables.tf:87]: Does the default of false match the current usage?

**Review by coufalja — DISMISSED** [review]

File: aws/cluster/variables.tf (hunk 1 above)
Comment (wandera-petr-cerny) [reply aws/cluster/variables.tf]: Hmm, so this is indirectly affected by `var.container-networking`? Might be worth reflecting this in the [spec](https://github.com/wandera/karch/pull/57/files#diff-2c55de97b5f7202e959655dfd70fee2a2b07a8a7e00ebee9140933ac4ee332ecR106)? 🤔  Not that important probably :-)

**Review by kuldazbraslav — DISMISSED** [review]

**Comment by jojand** [comment]
the change will produce this: 
```                  "kubeProxy":
                    "clusterCIDR": "100.96.0.0/11"
                    "cpuRequest": "100m"
              +     "enabled": false
                    "hostnameOverride": "@aws"
              -     "image": "gcr.io/google_containers/kube-proxy:1.15.12"
              +     "image": "gcr.io/google_containers/kube-proxy:v1.15.12"
                    "logLevel": 0
                  "kubeScheduler":
```
but it has no effect on the existing clusters - kube-proxy is not user right now

**Review by wandera-petr-cerny — APPROVED** [review]

**Comment by gerricchaplin** [comment]
Need to fix `image            = "gcr.io/google_containers/kube-proxy:v${var.kubernetes-version}" # From upstream` as its not always the case

**Pull request merged**